tower-http/src/sensitive_header.rs

```diff
@@ -47,103 +50,122 @@ use std::{
 use tower_layer::Layer;
 use tower_service::Service;
 
-/// Mark a header as [sensitive] on both requests and responses.
+/// Mark headers as [sensitive] on both requests and responses.
 ///
 /// Produces [`SetSensitiveHeader`] services.
 ///
 /// See the [module docs](crate::sensitive_header) for more details.
 ///
 /// [sensitive]: https://docs.rs/http/latest/http/header/struct.HeaderValue.html#method.set_sensitive
 #[derive(Clone, Debug)]
-pub struct SetSensitiveHeaderLayer {
-    header: HeaderName,
+pub struct SetSensitiveHeaderLayer<I> {
```

Comment:
take it or leave it: maybe worth renaming this to `SetSensitiveHeadersLayer` (and change all the other types correspondingly) now that it can set multiple headers?

Reply:
Good idea! I'll fix that when the clones have been dealt with 😊